http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfiguration.java

```diff
@@ -78,6 +78,12 @@
     /** The maximum number of times a failed request is retried. */
     int maxNumRetries();
 
+    /**
+     * The mode to use while processing requests. By default, requests are pinned to a URL until conditions are met
```

Comment:
`Indicates how the target node is selected for a given request.` (no need to say something about the default here, since it's always present)